appveyor.yml

```diff
@@ -0,0 +1,41 @@
+environment:
+  global:
+    TARGET: x86_64-pc-windows-msvc
+    RUST_VERSION: stable    
+    CRATE_NAME: bat
+    CARGO_HOME: "c:\\cargo"
+    RUSTUP_HOME: "c:\\rustup"
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init.exe -y --default-host %TARGET%
+  - set PATH=%PATH%;C:\cargo\bin
+  - rustc -Vv
+  - cargo -V
+  
+build: false
```

Comment:
Why is this set to `false`? It currently doesn't build `bat` or run the tests. If we want automatic deployment for Windows, we definitely need to enable continuous integration.